scripts/set_requester_pays.sh

```diff
@@ -1,15 +1,25 @@
 #!/bin/bash
 
 if (( $# < 1 )); then
-  echo "Usage: $0 TERRA_PROJECT_ID [PATH_TO_TERRA_BUCKET_LIST_FILE]"
-  echo "Unless you specify a source file, the script will read buckets from a file 'buckets.txt'."
+  echo "Usage: $0 TERRA_PROJECT_ID PATH_TO_TERRA_BUCKET_PATH"
+  echo "Unless you specify a source file or a string of buckets, the script will read buckets from a file 'buckets.txt'."
   echo "The source file must include newline-delimited Terra bucket paths of format gs://fc-XXXXX"
+  echo 'The string of Terra bucket paths can be formatted as "gs://fc-XXXXX gs://fc-XXXXX" or "fc-XXXXX fc-XXXXX"'
+  echo "i.e you can run `./set_requester_pays.sh project_id fc-12345`"
   echo "NOTE: this script requires you to be authed as your firecloud.org admin account."
   exit 0
-elif (( $# == 1 )); then
-  BUCKETS=$(cat buckets.txt)
-elif (( $# == 2 )); then
-  BUCKETS=$(cat $2)
+  elif (( $# == 1 )); then
+    BUCKETS=$(cat buckets.txt)
+  elif (( $# == 2 )); then
+    if [[ $2 == *"."* ]]; then
+      BUCKETS=$(cat $2)
+    else
+      BUCKETS=$2
+    fi
+  elif (( $# > 2 )); then
+    echo 'The Terra bucket paths must be formatted as "gs://fc-XXXXX gs://fc-XXXXX gs://fc-XXXXX"'
```

Comment:
this requirement is no longer true, right? we can tolerate a lack of `gs://` prefix now